src/standalone/Snow/Snow.jl

```diff
@@ -272,8 +328,15 @@ function ClimaLand.make_update_aux(model::SnowModel{FT}) where {FT}
 
         @. p.snow.T_sfc = snow_surface_temperature(p.snow.T)
 
-        @. p.snow.water_runoff =
-            compute_water_runoff(Y.snow.S, p.snow.q_l, p.snow.T, parameters)
+        p.snow.water_runoff .=
+            compute_water_runoff.(
+                Y.snow.S,
+                p.snow.q_l,
+                p.snow.T,
+                p.snow.ρ_snow,
+                snow_depth(model.parameters.density, Y, p, parameters),
```

Comment:
Question: can we always use snow_depth = SWE*rho_l/rho_snow? We just updated density above on line 319 using the density model correctly.

Reply:
This change was part of our consideration of the depth calculation when depth is directly available before making this change.